### PR TITLE
Add memcached support

### PR DESCRIPTION
**Pull request by shaer**
I added a memcached support which by testing reduced the memory usage and speeded up the caching process a little bit

### CODE REVIEW

**Comment by sc0ttkclark** [comment]
I was thinking about doing Memcached too, but was unable to find any hosts that had multiple memcached instances for us, this was to segment out just the Pods caching from the site cache. Memcached makes a great object cache so I left the Memcached solution out of the implementation since if it was going to be one or the other, it should be used for the site cache. By default, without Pods Alt Cache setup, it will use whatever the site uses.

Do you know of any configs that have multiple memcached instances for this use-case?

File: composer.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "description": "File-based or database-based caching solution for for hosts that have limitations on object caching.",
     "type": "wordpress-plugin",
     "keywords": ["pods", "wordpress", "cache", "wpengine"],
-    "homepage": "https://github.com/pods-framework/pods-alternative-cache",
+    "homepage": "https://github.com/shaer/pods-alternative-cache",
```
Comment (sc0ttkclark) [inline composer.json]: This line needs to be reverted

**Comment by sc0ttkclark** [comment]
Looks good, just needs that one tweak. Sorry for the delay here.

**Comment by shaer** [comment]
No problem at all, I have no clue why I have changed the homepage! :D , but I have reverted it back

Thanks

**Comment by sc0ttkclark** [comment]
You'll want to merge the latest master up to your repository as well, there are some conflicts right now.

**Pull request merged**